src/main.cpp

```diff
@@ -881,6 +878,9 @@ QString UsageArguments ( char **argv )
         "  -m, --htmlstatus      enable HTML status file, set file name\n"
         "  -o, --serverinfo      infos of this server in the format:\n"
         "                        [name];[city];[country as QLocale ID]\n"
+        "  -p, --port            set the local port number\n"
+        "  -Q, --qos             request quality of service in the IP header octet\n"
```

Comment:
Shouldn't it be p (port) < P (delaypan) < Q (qos) (i.e. -Q after -P)?

Reply:
Oops